src/app/pages/institutional-partnership/institutional-partnership.js

```diff
@@ -0,0 +1,35 @@
+import VERSION from '~/version';
+import {on} from '~/helpers/controller/decorators';
+import {Controller} from 'superb.js';
+import {description as template} from './institutional-partnership.html';
+
+export default class Institutional extends Controller {
+
+    init() {
+        this.template = template;
+        this.css = `/app/pages/institutional-partnership/institutional-partnership.css?${VERSION}`;
+        this.view = {
+            tag: 'main',
+            classes: ['institutional-page', 'page']
+        };
+    }
+
+    onLoaded() {
+        this.programTestimonial = this.el.querySelector('.testimonial-program');
+        this.applicationTestimonial = this.el.querySelector('.testimonial-application');
+        this.programTestimonial.classList.add('is-show');
```

Comment:
Apart from the top-level classes for the component, classes should be set in the view using state variables. In this case, you might have `this.whichTestimonialIsShowing` -- you'd probably choose a shorter name :) -- and your model would set variables according to it, like `this.model.testimonialShowClass = this.whichTestimonialIsShowing === 'application' ? 'is-show' : ''` and the template would have like `<div class="testimonial-application {model.testimonialShowClass}">`
If this doesn't make sense to you, put a TODO comment in here and I'll fix it up later since we're in a time crunch.